bin/boss-account.py

```diff
@@ -158,6 +159,9 @@ def __init__(self, bosslet_config):
         list = AlertList(args.bosslet_config)
 
     if args.create:
+        if list.create() is False:
```

Comment:
Why did you add this? if the list already exists it will always try to create it. These two lines mirror lines 168/169.

Reply:
I'll remove it.  I got into a catch 22 it created the topic but failed to create the alarms, then it would just not try and create them again.  It get it to run again I was changing a few lines.  I thought I had them all back but I guess I added this one by mistake.

Reply:
Got ya. I think what we need to do is to create the override `BillingList.exists()` so that it also checks to see if the alarms were created, and if not have it return false. Then in `BillingList.create()` have logic so that it doesn't try to create the SNS topic is it already exists.